src/worker/storage-models.ts

```diff
@@ -19,6 +19,14 @@ export interface StoredWordList {
    * The actual contents of said file.
    */
   wordlist: WordList;
+  /**
+   * Size of the file in bytes.
+   */
+  size?: number;
+  /**
+   * File extension.
+   */
+  type?: string;
```

Comment:
Please refer to #235.

To avoid scope-creep in this PR, feel free to **omit `type` entirely** and leave it for a future PR! 😅

Reply:
@eddieantonio #235 seems to be addressing the exact issue discussed here! I'll pick up on that issue once I wrap up this PR.